po/fr.po

```diff
@@ -176,11 +176,11 @@ msgstr "Mot de passe de protection"
 
 #: TwoFactorAuth/widgets/settings.py:93
 msgid "Auto-lock the application (m):"
-msgstr "Auto-verrouillage de l'application (m) :"
+msgstr "Auto-verrouillage de l'application (m):"
 
 #: TwoFactorAuth/widgets/settings.py:112
 msgid "Secret code generation time (s) :"
```

Comment:
@jayvdb You're right! the source strings sucks! if anyone of you wants to update it feel free to do so! Thanks